src/test/java/com/puppycrawl/tools/checkstyle/xpath/XpathQueryGeneratorTest.java

```diff
@@ -48,6 +50,26 @@
 
     private static FileText fileText;
 
+    private final CloseAndFlushTestByteArrayOutputStream outStream =
+            new CloseAndFlushTestByteArrayOutputStream();
```

Comment:
what this stream is for?

Reply:
This code is redundant, I will remove it. I added it while testing something. Thanks for pointing out.

Reply:
Done